.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -8,7 +8,7 @@ about: Suggest an idea for this project
 <!-- A clear and concise description of what the problem is. -->
 
 **Describe the solution you'd like**
-<!-- A clear and concise description of what you want to happen. Provide examples of use cases that this would enable and how it solves the problem described above. -->
+<!-- A clear and concise description of what you want to happen. Provide examples of real world use cases that this would enable and how it solves the problem described above. -->
 
 **Alternative Solutions**
 <!-- A clear and concise description of any alternative solutions or features you've considered. -->
```

Comment:
Again here, we should guide people towards more concrete examples. In particular I'd like to encourage people to explain how they are solving the problem *now*, as that would make it clearer just how costly it is to work around the lack of the proposed feature.